Core/HLE/proAdhoc.h

```diff
@@ -20,6 +20,7 @@
 // Net stuff
 #if defined(_WIN32)
 #include <WS2tcpip.h>
+#include "Common/CommonWindows.h"
```

Comment:
Can we move this include to only be in the cpp files?

We should avoid including CommonWindows in .h files whenever humanly possible.  This might be breaking armips.

-[Unknown]

Reply:
so does #undef min/max better? i need to use one of them to prevent getting an issue with std::min inside PSPNetconfDialog::DisplayMessage

Reply:
DisplayMessage is in a cpp file.  `#include "Common/CommonWindows.h"` is fine, but that line should be inside the file named `Core/Dialog/PSPNetconfDialog.cpp`.  Not inside Core/HLE/proAdhoc.h, since other cpp files are including that.

There are a few reasons to avoid Windows.h in other headers:
 * It slows down compilation of cpp files that don't need it.
 * Windows.h (even CommonWindows.h) defines a LOT of things, it's a kitchen sync.

-[Unknown]

Reply:
Ah i see. I was trying to fix it on the header file because it was the source of the std:min issue and was afraid it will cause similar issue in the future when imported to other cpp like PSNetconfDialog.cpp did :) without importing net/adhoc header  PSNetconfDialog.cpp doesn't have issue with std::min